pkg/test/go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/anz-bank/sysl-catalog/pkg/test
+
+go 1.14
+
+require (
+	github.com/anz-bank/protoc-gen-sysl v0.0.8 // indirect
+	github.com/anz-bank/sysl v0.97.0
+	github.com/anz-bank/sysl-catalog v1.4.29
+	github.com/sirupsen/logrus v1.6.0
+	github.com/spf13/afero v1.2.2
+	github.com/stretchr/testify v1.5.1
+)
```

Comment:
get rid of this file

Reply:
It has to be there for sysl modules to work sadly :(

Reply:
why?

Reply:
you've given 0 explanation to any of the changes in here, so some elaboration would be nice

Reply:
Sure, so sysl modules uses go modules under the hood to fetch remote sysl files. In order for that to work it needs a go.mod file (https://github.com/anz-bank/sysl/blob/0275e2741ea191f02e42dc1d3c8166a456cd7899/pkg/mod/module.go#L47)

Not exactly sure why the go.mod file is needed for go get to work, but I can ask @ChloePlanet for more details if you'd like (Or dig a little deeper myself).